README.md

```diff
@@ -1,3 +1,143 @@
 # smpte.js
 
-Easily deal with Timecode SMPTE format
+Easily deal with Timecode SMPTE format.
+
+## Installation
+
+Using `npm` or `yarn`:
+```
+npm install smpte.js
+
+yarn add smpte.js
+```
+
+## Usage
+
+### Example
+Using `ES6` sintax:
+
+```javascript
+import SMPTE from 'smpte.js';
+
+SMPTE.defaults.frameRate = 29.97;
+SMPTE.defaults.dropFrame = true;
+
+let timecode = new SMPTE(1);
+let timecodeToAdd = new SMPTE('00;00;00;02');
+let anotherTimecodeToAdd = SMPTE.fromSeconds(1);
+
+console.log(
+    timecode.add(timecodeToAdd)
+    .add(anotherTimecodeToAdd)
+    .toString('00:00:00;00')
+);
+// 00:00:01;03
+```
+
+### Object Instantiation
+SMPTE objects can be instatianted from frame count, [date object](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Date), seconds, timecode integer parts or directly from timecode string.
+
+```javascript
+// Frame count
+let tm = SMPTE.fromFrames(1440, 24, false);
+
+// Date object
+let tm = SMPTE.fromDate(Date.now(), 24, false);
+
+// Seconds
+let tm = SMPTE.fromSeconds(60, 24, false);
+
+// Timecode integer parts
+let tm = SMPTE.fromParts(0, 1, 0, 0, 24, false);
+
+// Timecode string
+let tm = SMPTE.fromTimecode('00:01:00:00', 24, false);
+
+// or using the new operator
+let tm = new SMPTE('00:01:00:00', 24, false);
+```
+
+### Properties
+
+SMPTE objects provide the following properties once an object is created:
+
+| Property               | Type      | Description                  |
+| ---------------------- | --------- | ---------------------------- |
+| `framecount`           | `Number`  | Total number of frames       |
+| `hours`                | `Number`  | Hours number                 |
+| `minutes`              | `Number`  | Minutes number               |
+| `seconds`              | `Number`  | Seconds number               |
+| `frames`               | `Number`  | Frames number                |
+| `durationInSeconds`    | `Number`  | Timecode duration in seconds |
+| `attributes.df`        | `Boolean` | Drop frame indicator         |
+| `attributes.frameRate` | `Number`  | Frame rate in FPS            |
+
+### Methods
+
+Each SMPTE object provides the following methods:
+
+#### `addFromSeconds(seconds)` - [**chainable**](#chainable)
```

Comment:
I think we don't need to explain what chainable is, just add some examples on how to use it. We can also add what the method will return.